pkg/network/protocols/events/consumer_test.go

```diff
@@ -208,11 +209,13 @@ func newEBPFProgram(c *config.Config) (*manager.Manager, error) {
 		},
 	}
 
-	Configure(config.New(), "test", m, &options)
+	ddEbpfManager := ddebpf.NewManager(m, "usm", &ebpftelemetry.ErrorsTelemetryModifier{})
```

Comment:
do you really need the errorstelemetrymodifier for the test?

Reply:
@val06  If the ebpf code uses the telemetry macros, the modifier is required to patch the code correctly.